vscode_extension/vsc-extension-quickstart.md

```diff
@@ -0,0 +1,42 @@
+# Welcome to your VS Code Extension
```

Comment:
We should delete this file.

Reply:
Agreed. Done. Nothing of value worth keeping in the auto-generated doc.